repose-aggregator/components/services/uri-redaction-service/uri-redaction-service-impl/src/main/scala/org/openrepose/core/services/uriredaction/UriRedactionServiceImpl.scala

```diff
@@ -0,0 +1,90 @@
+/*
+ * _=_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=
+ * Repose
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Copyright (C) 2010 - 2015 Rackspace US, Inc.
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
+ */
+package org.openrepose.core.services.uriredaction
+
+import com.typesafe.scalalogging.slf4j.StrictLogging
+import javax.annotation.{PostConstruct, PreDestroy}
+import javax.inject.{Inject, Named}
+import org.openrepose.commons.config.manager.UpdateListener
+import org.openrepose.core.services.config.ConfigurationService
+import org.openrepose.core.services.uriredaction.UriRedactionServiceImpl._
```

Comment:
I think we normally stick this in the class itself. I would suggest we do that here for the sake of consistency.

As for whether or not we should do that everywhere, I haven't really formulated a strong opinion. I don't think the Scala style guide is explicit about it, but it does import the companion into the class in the example:
https://docs.scala-lang.org/tour/singleton-objects.html#companion-objects

Reply:
This `import` was moved inside the class definition.